### PR TITLE
Emergency Spell - March 24, 2023

### PR DESCRIPTION
**Pull request by prose11**

### CODE REVIEW

File: governance/votes/Executive vote - March 24, 2023.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: Template - [Executive Vote] Emergency Spell for Shutdown Implications - March 24, 2023
+summary: Enables the Emergency Shutdown Module to remove Governance’s authority on the MCD_JOIN_CRVV1ETHSTETH_A, MCD_CROPPER, and CHANGELOG contracts to prevent malicious actions after shutdown.
+date: 2023-03-24T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Emergency Spell for Shutdown Implications - March 24, 2023
```
Comment (astronautthis) [inline governance/votes/Executive vote - March 24, 2023.md:8]:
```suggestion
# [Executive Proposal] Emergency Spell to resolve Emergency Shutdown Governance Vulnerability - March 24, 2023
```

File: governance/votes/Executive vote - March 24, 2023.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+---
+
+## Proposal Details
+
+### Emergency Shutdown Module Changes
+
+Based on this [recommendation](https://forum.makerdao.com/t/emergency-shutdown-governance-vulnerability-proposed-emergency-spell/20255) from the [Protocol Engineering Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP7), the [Emergency Shutdown Module](https://manual.makerdao.com/governance/emergency-shutdown) will be able to remove Governance’s authority over the following contracts, if this executive proposal passes.
+
+#### Effected Contracts
```
Comment (astronautthis) [inline governance/votes/Executive vote - March 24, 2023.md:35]:
```suggestion
#### Affected Contracts 
```

Comment (prose11) [reply governance/votes/Executive vote - March 24, 2023.md:35]: I had it like this but retro convinced me otherwise, state your case ser

Comment (prose11) [reply governance/votes/Executive vote - March 24, 2023.md:35]: Pablo is a grammar king and agrees, will stop listening to retro lol (only for grammar <3)

File: governance/votes/Executive vote - March 24, 2023.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+[MCD_CROPPER](https://etherscan.io/address/0x8377cd01a5834a6ead3b7efb482f678f2092b77e) -
+
+This contract is upgradable and authorized on MCD_JOIN_CRVV1ETHSTETH_A, so it could be used to upgrade that contract post-shutdown and execute the attack mentioned above.
+
+[CHANGELOG](https://etherscan.io/address/0xda0ab1e0017debcd72be8599041a2aa3ba7e740f) - 
+
+While there is no known danger to the core system from this contract after shutdown, it is being included because external integrations may be depending on it in unsafe ways, and knowing it cannot be corrupted may be useful for future tooling built around Emergency Shutdown.
+
+Specifically, this means that after shutdown is triggered, it will be possible to permissionlessly de-authorize the GSM pause proxy from these contracts. This will need to be done within the GSM delay. It is a pattern in use for a number of other contracts already.
```
Comment (astronautthis) [inline governance/votes/Executive vote - March 24, 2023.md:49]: If I understood the change correctly, I think this paragraph should be earlier, where you say "the [Emergency Shutdown Module](https://manual.makerdao.com/governance/emergency-shutdown) will be able to remove Governance’s authority over the following contracts".

Comment (prose11) [reply governance/votes/Executive vote - March 24, 2023.md:49]: I agree, good suggestion

File: governance/votes/Executive vote - March 24, 2023.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+## Proposal Details
+
+### Emergency Shutdown Module Changes
+
+Based on this [recommendation](https://forum.makerdao.com/t/emergency-shutdown-governance-vulnerability-proposed-emergency-spell/20255) from the [Protocol Engineering Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP7), the [Emergency Shutdown Module](https://manual.makerdao.com/governance/emergency-shutdown) will be able to remove Governance’s authority over the following contracts, if this executive proposal passes.
+
+#### Effected Contracts 
+
+[MCD_JOIN_CRVV1ETHSTETH_A](https://etherscan.io/address/0x82d8bfdb61404c796385f251654f6d7e92092b5d) -
```
Comment (astronautthis) [inline governance/votes/Executive vote - March 24, 2023.md:39]:
```suggestion
[MCD_JOIN_CRVV1ETHSTETH_A](https://etherscan.io/address/0x82d8bfdb61404c796385f251654f6d7e92092b5d) 
```

Comment (astronautthis) [reply governance/votes/Executive vote - March 24, 2023.md:39]: Same for the two below, not sure if the hifens were intentional. If so, please ignore

Comment (prose11) [reply governance/votes/Executive vote - March 24, 2023.md:39]: seemed more necessary before I combines the text and link, will remove

**Comment by prose11** [comment]
Ty @astronautthis !

**Pull request merged**